src/layouts/sidebar/index.tsx

```diff
@@ -140,14 +182,18 @@ export default function Sidebar() {
               width: 'inherit',
             }}
           >
-            <Button
-              variant='outlined'
-              sx={{ color: '#ffffff' }}
-              endIcon={<LogoutIcon />}
-              onClick={onLogout}
-            >
-              Logout
-            </Button>
+            {window.location.pathname === ROUTE.LOGIN ? (
```

Comment:
Do we really need this condition? Sidebar should always mounted only for components where we need it. So if we don't need the sidebar we can avoid using the withLayout HOC for it

Reply:
okay